<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerWnd</class>
 <widget class="QWidget" name="CustomerWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customers</string>
  </property>
  <property name="windowIcon">
   <iconset resource="picture.qrc">
    <normaloff>:/build-course-Desktop_Qt_6_5_3_MinGW_64_bit-Debug/icons/iconka.ico</normaloff>:/build-course-Desktop_Qt_6_5_3_MinGW_64_bit-Debug/icons/iconka.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}
QTableView {
	font: 9pt &quot;Century Gothic&quot;;
    border: 3px solid black;
    border-radius: 10px;
    background-color: #f0f0f0;  /* Фон заокругленого віджета */
    selection-background-color: #a6a6a6;  /* Колір виділення */
}
/* Імена стовбців */
QHeaderView::section {
    background-color: #404040;  /* Колір фону */
    color: white;  /* Колір тексту */
    padding: 4px;
    border: 1px solid #303030;  /* Границі */
}

/* Номерація рядків */
QTableView QTableCornerButton::section {
    background-color: #404040;  /* Колір фону */
    color: white;  /* Колір тексту */
    border: 1px solid #303030;  /* Границі */
}


QTableView::item:hover {
    border: 2px solid #505050;  /* Колір границі при наведенні */
}

QTableView::item:selected {
    border: 2px solid #303030;  /* Колір границі при виділенні */
}
QLineEdit {
font: 12pt &quot;Candara&quot;;
                 background-color: white;
                 color: black;
                 border: 1px solid rgb(110, 110, 110);
              border-radius: 10px;
                 padding: 3px 7px;
             }

             QLineEdit:focus {
                 border: 2px solid rgb(90, 90, 90);
            }
QPushButton
{
	font: 12pt &quot;Candara&quot;;
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
border-radius: 20px;
box-shadow: 0 0 20px 10px rgb(240, 158, 255);
width: 50px;
  height: 40px;
}
QPushButton:hover {
               border: 2px solid rgb(200, 200, 200);
            }
QPushButton:pressed {
             background-color: rgb(255, 255, 255);
	        color: rgb(0, 0, 0);
            }
QLabel{font: 12pt &quot;Candara&quot;;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="searchLbl">
       <property name="styleSheet">
        <string notr="true">QLabel {
    border: 1px solid #000000; /* Задайте границю */
    border-top-left-radius: 15px; /* Заокруглення тільки з одного боку */
    border-bottom-left-radius: 15px;
    border-top-right-radius: 0; /* Встановіть інші кути без заокруглення */
    border-bottom-right-radius: 0;
    background-color: rgb(255, 255, 255); /* Колір фону */
    color: #000000; /* Колір тексту */
    padding: 5px; /* Відступи */
    width: 35px;
    height: 30px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="picture.qrc">:/photo/search.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchLE">
       <property name="styleSheet">
        <string notr="true"> border: 1px solid #000000; /* Задайте границю */
    border-top-left-radius: 0px; /* Заокруглення тільки з одного боку */
    border-bottom-left-radius: 0;
    border-top-right-radius: 15px; /* Встановіть інші кути без заокруглення */
    border-bottom-right-radius:15px;
    background-color: rgb(255, 255, 255); /* Колір фону */
    color: #000000; /* Колір тексту */
    padding: 5px; /* Відступи */
    width: 665px;
    height: 30px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="nameLb">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="nameLE"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="priceLb">
     <property name="text">
      <string>Price:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="priceLE"/>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="bodyPb">
     <property name="text">
      <string>Body</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="surenameLb">
     <property name="text">
      <string>Surename:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="surenameLE"/>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="bodyLb">
     <property name="text">
      <string>Body:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="bodyLE"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="numberLb">
     <property name="text">
      <string>Number:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="numberLE"/>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="masterLb">
     <property name="text">
      <string>Master:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="masterLE"/>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="addPb">
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTableView" name="customerTv"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLE</tabstop>
  <tabstop>surenameLE</tabstop>
  <tabstop>numberLE</tabstop>
  <tabstop>priceLE</tabstop>
  <tabstop>bodyLE</tabstop>
  <tabstop>masterLE</tabstop>
  <tabstop>addPb</tabstop>
  <tabstop>bodyPb</tabstop>
  <tabstop>customerTv</tabstop>
 </tabstops>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
